src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -26,6 +26,15 @@ cd "%BUILDTOOLS_PACKAGE_DIR%\tool-runtime\"
 call "%DOTNET_CMD%" restore --source https://www.myget.org/F/dotnet-core/ --source https://www.myget.org/F/dotnet-buildtools/ --source https://www.nuget.org/api/v2/
 call "%DOTNET_CMD%" publish -f dnxcore50 -r %BUILDTOOLS_TARGET_RUNTIME% -o "%TOOLRUNTIME_DIR%"
 
+:: Temporary workaround for CLI #710: copy native artifacts that weren't during dotnet publish.
+set TOOL_RUNTIME_PACKAGE_DIR=%BUILDTOOLS_PACKAGE_DIR%\tool-runtime\packages
+call "%DOTNET_CMD%" restore --source https://www.myget.org/F/dotnet-core/ --source https://www.myget.org/F/dotnet-buildtools/ --source https://www.nuget.org/api/v2/ --packages "%TOOL_RUNTIME_PACKAGE_DIR%"
+for /f %%p in ('dir "%TOOL_RUNTIME_PACKAGE_DIR%" /B /A:D') do (
```

Comment:
This is actually only a problem for the MSBuild and Microsoft.Build.Targets packages, can we constrain the workaround for only those packages?